### ISSUE TITLE
[BUG] Gallery builder photo scroll

### ISSUE THREAD

**Issue by barangungor** [fireslime/photo_view#345]
I have difficulty scrolling to change photos. I have no problem changing photos with a mouse in the emulator, but when I run it on my phone, I have problems with the changes.

**Comment by abdalmonem** [comment]
same thing with me , i notice this after upgrade to 0.10.2

**Comment by abbas-25** [comment]
same issue!!

**Comment by andreystavitsky** [comment]
Same issue

**Comment by nbonamy** [comment]
Same issue here. This is fixed in https://github.com/fireslime/photo_view/commit/74e5a878f99e72dfbe76dfce61c76d037e96d3db. So until 0.10.3 is released, you can override and fetch from HEAD by adding this to your pubspec.yaml:

```
dependency_overrides:
  photo_view:
    git:
      url: https://github.com/fireslime/photo_view
```

**Comment by github3t** [comment]
I had the same problem and can confirm that the latest HEAD fixes it for me.

**Comment by renancaraujo** [comment]
Sorry for that delay, we gonna have a minor version today.

**Comment by renancaraujo** [comment]
Should be fixed on 0.10.3
